cyctl/internal/get/modules.go

```diff
@@ -47,16 +47,34 @@ func listModules(clientset *client.CyclopsV1Alpha1Client, moduleNames []string)
 		for _, name := range moduleNames {
 			nameSet[name] = struct{}{}
 		}
-		filteredModules = []v1alpha1.Module{}
+		foundModules := []v1alpha1.Module{}
+		notFoundModules := []string{}
```

Comment:
Could you initialize these with `make`?
```suggestion
		foundModules := make([]v1alpha1.Module, 0)
		notFoundModules := make([]string, 0)
```

Reply:
Done. Also added the exit status.